### DISCUSSION TITLE
What is each column in two csv file, detailed description  thx

### DISCUSSION THREAD

**Discussion by iorileslie** [gempy-project/gempy#845] [Q&A]
X | Y | Z | azimuth | dip | polarity | formation

X | Y | Z | formation

And what is the relationship between two csv files?

**Comment by AlexanderJuestel** [comment]
@iorileslie, I converted your issue to a discussion. 

Please see the documentation and in particular, this tutorial where this is elaborated on: https://docs.gempy.org/tutorials/index.html#chapter-1-gempy-fundamentals

The X, Y, Z coordinates are the location of your input data, formation the rock unit the point belongs to and dip/azimuth are the orientation values needed to calculate the scalar field

**Reply by iorileslie** [reply]
OK,thanks. Looks like I need to read the documentation carefully ^v^
